Detections/MultipleDataSources/CrossCloudUnauthorizedCredentialsAccessDetection.yaml

```diff
@@ -0,0 +1,149 @@
+      case (
+        Severity >= 7.0, "High",
+        Severity between (4.0 .. 6.9), "Medium",
+        Severity between (1.0 .. 3.9), "Low",
+        "Unknown")
+        | distinct
+            AWSAlertTime,
+            ActivityType,
+            AWSAlertId,
+            AWSAlertLink,
+            AWSAlertTitle,
+            AWSAlertDescription,
+            AWSresourceType,
+            Arn,
+            Severity,
+            RDSactionType,
+            RDSApplication,
+            RDSInstanceId,
+            RDSUser,
+            AWSNetworkEntity
+        );
+      // Define variable 'Azure_sigin' to collect Azure portal sign-in activities
+      let Azure_sigin = materialize (
+          SigninLogs
+          | where AppDisplayName == "Azure Portal"
+          | where isnotempty(OriginalRequestId)
+          | summarize 
+              AzureSuccessfulEvent = countif(ResultType == 0), 
+              AzureFailedEvent = countif(ResultType != 0), 
+              totalAzureLoginEventId = dcount(OriginalRequestId), 
+              AzureFailedEventsCount = dcountif(OriginalRequestId, ResultType != 0), 
+              AzureSuccessfuleventsCount = dcountif(OriginalRequestId, ResultType == 0),
+              AzureSetOfFailedevents = makeset(iff(ResultType != 0, OriginalRequestId, ""), 5), 
+              AzureSetOfSuccessfulEvents = makeset(iff(ResultType == 0, OriginalRequestId, ""), 5) 
+              by 
+              IPAddress, 
+              UserPrincipalName, 
+              bin(TimeGenerated, 1min), 
+              UserAgent,
+              ConditionalAccessStatus,
+              OperationName,
+              RiskDetail,
+              AuthenticationRequirement,
+              ClientAppUsed
+          // Extracting the name and UPN suffix from UserPrincipalName
+          | extend
+              Name = tostring(split(UserPrincipalName, '@')[0]),
+              UPNSuffix = tostring(split(UserPrincipalName, '@')[1])
+          );
+      // Join 'AwsAlert' and 'Azure_sigin' on the AWS Network Entity and Azure IP Address
+      AwsAlert
+      | join kind=inner Azure_sigin on $left.AWSNetworkEntity == $right.IPAddress
+entityMappings:
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: IPAddress
+  - entityType: Account
+    fieldMappings:
+      - identifier: Name
+        columnName: Name
+      - identifier: UPNSuffix
+        columnName: UPNSuffix
+customDetails:
+  AWSAlertUserName: RDSUser
+  AWSArn: Arn
+  AWSresourceType: AWSresourceType
+  AWSInstanceType: RDSactionType
+  AWSAplicationName: RDSApplication
+  AWSInstanceId: RDSInstanceId
+  AzureUserAgent: UserAgent
+  AzureUser: UserPrincipalName
+  AzureClientAppUsed: ClientAppUsed
+  AzConditionalAccess: ConditionalAccessStatus
+  AzureOperationName: OperationName
+  AzureRiskDetail: RiskDetail
+  AzAuthRequirement: AuthenticationRequirement
+  alertSeverity: Severity
+alertDetailsOverride:
+  alertDisplayNameFormat: "{{AWSAlertTitle}}"

```

Comment:
Do we completely want to replace the alert name with that? I would think it might be better to be:

"IP address {{IPAddress}} in {{AWSAlertTitle}} seen in Azure Signin Logs with {{UserPrincipalName}}"

Reply:
Updated As per your suggestion